README.md

```diff
@@ -83,6 +83,9 @@ The relayer is configured via a JSON file, the path to which is passed in via th
   `"supported-destinations": []string` 
   - List of destination subnet IDs that the source subnet supports. If empty, then all destinations are supported.
 
+  `"catch-up-block-height": integer`
+  - The block height at which to back-process transactions from the source subnet. If the database already contains a later block height for the source subnet, then that will be used instead.
+
```

Comment:
I've gone back and forth a bit on whether I think specifying a height of a max number of blocks to process would be better here. I think for release, being able to set the height of the warp upgrade makes the most sense, but in the future, setting a max number of blocks might make more sense.

Reply:
That's an interesting idea. Especially since processing from height 0 is perfectly valid, specifying a limit (and having a default value) would be helpful. Created an issue to track this: https://github.com/ava-labs/awm-relayer/issues/127